ImageFeed/Auth/WebViewViewController.swift

```diff
@@ -0,0 +1,97 @@
+import UIKit
+import WebKit
+
+final class WebViewViewController: UIViewController{
```

Comment:
```suggestion
final class WebViewViewController: UIViewController {
```

Давай такие помарки проверишь по всему проекту и поправишь если остались еще?

Reply:
Осталось не исправленным